### ISSUE TITLE
Typo in file name collision message

### ISSUE THREAD

**Issue by jhlegarreta** [nextflow-io/nextflow#1036]
## Bug report 

The [name collision log debug](https://github.com/nextflow-io/nextflow/blob/master/modules/nextflow/src/main/groovy/nextflow/processor/TaskProcessor.groovy#L1784) in  the [TaskProcessor.groovy file](https://github.com/nextflow-io/nextflow/blob/master/modules/nextflow/src/main/groovy/nextflow/processor/TaskProcessor.groovy) has [recently been improved](3def8334a01c0953af6ab9aff7bced19fcb9d4ea).

Although the collision error message unequivocal due to the `input file name collision` part of the subsequent message, I'd dare to say that `staing` is either missing some consonant/has some typo, or the verb was intended to be another one, since AFAIK `staing` is not a verb form.

### Expected behavior and actual behavior

Expected: have a typo-free error message.
Actual: the message verb has some typo. A few suggestions for the fix:
```
 log.debug("Process $name > collision check for file names: $allNames")
```
```
 log.debug("Process $name > collision check issue detected for file names: $allNames")
```

### Steps to reproduce the problem

Have multiples files for the same file processed by nextflow (actually I cannot see the name collision, so still finding where it comes from).

### Program output 

```
 DEBUG nextflow.processor.TaskProcessor - Process Tracking > collision check staing file names: [tracking.trk:9]
```

### Environment 

* Nextflow version: version 0.31.1 build 4886
* Java version:
java version "1.8.0_181"
Java(TM) SE Runtime Environment (build 1.8.0_181-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.181-b13, mixed mode)

* Operating system: Ubuntu 16.04 LTS

### Additional context

Very low priority. Easy to fix. Thanks for bringing this amazing tool to reproducible science.

**Comment by jhlegarreta** [comment]
I can do a PR if the appropriate message is decided from the toolkit maintainers.

**Comment by pditommaso** [comment]
Yeah, it should be *staging*. Already fixed it. Thanks for reporting it.

**Comment by jhlegarreta** [comment]
Awesome @pditommaso . Thanks for this invaluable tool.

**Comment by pditommaso** [comment]
Thanks ;)

**Comment by pditommaso** [comment]
Fixed by 6e2b6e47f9d84abf0c0d4922fb6eeb8bb0b56ade
